website_sale_price_tier/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+product_uom_portal_read_access,Product UoM - Portal Read Access,product.model_product_uom,base.group_portal,1,0,0,0
```

Comment:
Why do you need portal access? This is for website visitors only.

Reply:
I commented on this in the v9 PR.